report-service/src/main/java/gov/cdc/dataingestion/nbs/ecr/constant/CdaConstantValue.java

```diff
@@ -0,0 +1,60 @@
+package gov.cdc.dataingestion.nbs.ecr.constant;
+
+public class CdaConstantValue {
+    private CdaConstantValue() {
+
+    }
+    public static final String ROOT_ID = "2.16.840.1.113883.19";
+    public static final String CODE_SYSTEM = "2.16.840.1.113883.6.1";
+    public static final String CODE_SYSTEM_NAME = "LOINC";
+    public static final String XML_NAME_SPACE_HOLDER = "urn:hl7-org:v3";
+    public static final String NAME_SPACE_URL = "http://www.w3.org/2001/XMLSchema-instance";
+    public static final String STATE = "STATE";
+    public static final String COUNTY = "COUNTY";
+    public static final String COUNTRY = "COUNTRY";
+    public static final String USESABLE_PERIOD = "useablePeriod";
+    public static final String CODE = "123-4567";
+    public static final String CLINICAL_CODE_SYSTEM = "Local-codesystem-oid";
+    public static final String CLINICAL_CODE_SYSTEM_NAME = "LocalSystem";
+    public static final String CODE_DISPLAY_NAME = "Interested Parties Section";
+    public static final String CLINICAL_TITLE = "INTERESTED PARTIES SECTION";
+    public static final String ACT_CODE_DISPLAY_NAME = "Interested Party";
+    public static final String NOT_FOUND_VALUE = "NOT_FOUND";
+    public static final String ID_ROOT = "2.16.840.999999";
```

Comment:
Can this values change in the future? if so, can this be moved to config file?

Reply:
I believe this won't be changed in the future as it these were hard coded in Rhapsody xml mapper script, The reason it is has it own constant class because these value are being used in multiple places